tpp-app/App.js

```diff
@@ -45,7 +47,16 @@ export default function App() {
       <NavigationContainer>
         <Tab.Navigator>
           <Tab.Screen name="Info" component={Info} options={{
-            headerShown: false
+            headerShown: false,
+            tabBarIcon: ({tintColor}) => (
+            <View style={{top: 3}}>
+                  <Image
+                    source={InfoIcon}
+                    style={{width: 20, height: 20, tintColor: tintColor}}
+                  />
+            </View>
```

Comment:
You should pull this out and create its own React Component so that there is less in-line code. Similar to how CustomTabBarButton is done below.

Reply:
Addressed! Thanks for the tip